### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by birdofpreyru** [birdofpreyru/babel-plugin-react-css-modules#2]
A naive attempt to update current NPM dependencies to latest versions fails miserably: it breaks flow tests (because of breaking change in https://github.com/facebook/flow/blob/master/Changelog.md#01320 (disallow using any-typed values type annotations); it also breaks the build because of some issue with source maps generation.

Thus, it needs some time to do the update properly.

**Comment by birdofpreyru** [comment]
`@babel/cli@7.11.5` contains a bug, which breaks the build: https://github.com/babel/babel/issues/12025
Keeping `v7.10.5` for now.

**Comment by birdofpreyru** [comment]
As of v6.0.4, everything beside `@babel/cli` is updated to the latest versions.

**Comment by dxhuii** [comment]
When to support postcss version 8.x .
Now, if you want to use postcss-nested version 5.0, you will be prompted to upgrade to postcss version 8.0

**Comment by birdofpreyru** [comment]
@dxhuii I have not noticed it requires an update until your message. After a brief look, it seems to require some updates in the plugin code. I'll try to do it this week, depending on how tough the changes are, and how much time I have.

**Comment by dxhuii** [comment]
ok

**Comment by dxhuii** [comment]
After the update, postcss-nested@5.0 is supported, but not postcss-scss@3.0.1, reporting the following error.
```bash
ERROR in ./src/app/components/Detail/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: /Users/dingxiaohui/Documents/web/react-5ddm/src/app/components/Detail/index.js: Could not resolve the styleName 'detail-mini'.
    at handleError (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@dr.pogodin/babel-plugin-react-css-modules/dist/getClassName.js:18:11)
    at /Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@dr.pogodin/babel-plugin-react-css-modules/dist/getClassName.js:103:14
    at Array.map (<anonymous>)
    at _default (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@dr.pogodin/babel-plugin-react-css-modules/dist/getClassName.js:83:6)
    at _default (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@dr.pogodin/babel-plugin-react-css-modules/dist/resolveStringLiteral.js:20:55)
    at PluginPass.JSXElement (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@dr.pogodin/babel-plugin-react-css-modules/dist/index.js:191:47)
    at newFn (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@babel/traverse/lib/visitors.js:175:21)
    at NodePath._call (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit (/Users/dingxiaohui/Documents/web/react-5ddm/node_modules/@babel/traverse/lib/path/context.js:92:31)
 @ ./src/app/components/Vod/index.js 12:37-67
 @ ./src/app/components/SideBar/index.js
 @ ./src/app/pages/subject/news/index.js
 @ ./src/app/router/list.js
 @ ./src/app/router/index.js
 @ ./src/client/index.js
 @ multi webpack-hot-middleware/client?path=http://localhost:4001/__webpack_hmr @babel/polyfill ./src/client/index
```

The support for postcss-scss@^2.1.1 is fine, no problem. The above error is reported as soon as it is updated.

You have time to look at it, I'm not really sure, it's a problem with this package you have.

**Comment by dxhuii** [comment]
I saw today that postcss has been upgraded to 8.1.1, I wonder if other dependencies have been upgraded. Began to report errors again, it feels like this upgrade is really endless, has been the problem, not across the big version, surprisingly also have problems.

**Comment by birdofpreyru** [comment]
Yeah @dxhuii , they released PostCSS 8.0.x with a bug in the new AST walking logic, which behaved simlar to 7.x. They fixed it in 8.1.x, and it became a breaking change which was supposed to be a part of 8.0.x release.

I just patched this plugin, **v6.0.6** should work fine with PostCSS 8.1.x (I am still to double-check it in a real project, and if I find some problems, I'll patch it further shortly).

**Comment by dxhuii** [comment]
Ok, I'm testing so far with no problems.
